cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1329,7 +1329,7 @@ Slice::Gen::ForwardDeclVisitor::visitEnum(const EnumPtr& p)
         //
         if (explicitValue)
         {
-            H << " = " << int64ToString((*en)->value());
+            H << " = " << std::to_string((*en)->value());
```

Comment:
We can probably remove the `to_string` call here, `<<` operator should handle `int64_t`